src/Toggle/Toggle.js

```diff
@@ -66,6 +76,32 @@ function getStyles(props, context, state) {
     },
   };
 
+  Object.assign(styles.track,
+    trackStyle,
+    state.switched && styles.trackWhenSwitched,
+    state.switched && trackSwitchedStyle,
+    disabled && styles.trackWhenDisabled
+  );
+
+  Object.assign(styles.thumb,
+    thumbStyle,
+    state.switched && styles.thumbWhenSwitched,
+    state.switched && thumbSwitchedStyle,
+    disabled && styles.thumbWhenDisabled
+  );
+
+  if (state.switched) {
+    styles.thumb.marginLeft = 0 - styles.thumb.width;
+  }
+
+  Object.assign(styles.icon, iconStyle);
```

Comment:
@oliviertassinari okay, should I roll back the styling to`render()`? Reapply this PR to `next`? Please help me to get this PR merged.